test/1_stdlib/UnsafePointer.swift.gyb

```diff
@@ -389,5 +389,26 @@ UnsafeMutablePointerTestSuite.test("customPlaygroundQuickLook") {
   }
 }
 
-runAllTests()
+% for (SelfName, SelfType) in [
+%         ('UnsafePointer', 'UnsafePointer<Int>'),
+%         ('UnsafeMutablePointer', 'UnsafeMutablePointer<Int>')]:
+
+${SelfName}TestSuite.test("Comparable") {
+  let instances = [
+    0x00000001,
+    0xFF00FF00,
+    0x00001111,
+    0x00000002,
+    0xFFFFFFFF,
+    0x00001111
+  ].map { ($0, ${SelfType}(bitPattern: $0)!) }
```

Comment:
It would be good to throw in some 64-bit values if we are on a 64-bit platform (see #ifs in tests above).